CA_DataUploaderLib/IOconf/IOConfFileLoader.cs

```diff
@@ -29,21 +29,23 @@ public static class IOconfFileLoader
             ("Node", (r, l) => new IOconfNode(r, l)),
         };
 
-        public static (string, IEnumerable<IOconfRow>) Load(List<IOconfRow> target)
+        public static (List<string>, IEnumerable<IOconfRow>) Load()
         {
             if (!File.Exists("IO.conf"))
-                throw new Exception($"Could not find the file {Directory.GetCurrentDirectory()}\\IO.conf");
+            {
+                throw new Exception($"Could not find the file {Directory.GetCurrentDirectory() + "\\IO.conf"}");
```

Comment:
these are equivalent (the earlier is the idomatic version for string interpolation):
{Directory.GetCurrentDirectory()}\\\\IO.conf
{Directory.GetCurrentDirectory() + "\\\\IO.conf"